src/scripts/components/story-header/story-header.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {FunctionComponent} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {Link} from 'react-router-dom';
+import {useSelector} from 'react-redux';
+
+import {storyListSelector} from '../../selectors/story/list';
+
+import {StoryMode} from '../../types/story-mode';
+
+import styles from './story-header.styl';
+
+interface Props {
+  mode: StoryMode;
+  storyIds: string[];
+}
+
+const StoryHeader: FunctionComponent<Props> = ({mode, storyIds}) => {
+  const storyList = useSelector(storyListSelector);
+  const storyId = storyIds[0];
```

Comment:
Why is the index always `0`? And don't we have the currently selected story already in the story component? Can't we just pass the title to the story header component?